src/main/java/Duke.java

```diff
@@ -1,10 +1,266 @@
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    private static final String FILEPATH = "./data/tasks.json";
+    private static Duke.Storage storage = new Duke.Storage(FILEPATH);
+    private static final String tab = "      ";
+    private static final String line = "------------------------------------------------------------";
+
+    public static void textBox(String... messages) {
+
+        System.out.println(tab + line);
+        for (int i = 0; i < messages.length; i++) {
+            System.out.println(tab + " " + messages[i]);
+        }
+        System.out.println(tab + line);
+    }
+
+    public static void main(String[] args) throws NoSuchTaskException, InvalidInputException, FileNotFoundException {
+//        String logo = " ____        _        \n"
+//                + "|  _ \\ _   _| | _____ \n"
+//                + "| | | | | | | |/ / _ \\\n"
+//                + "| |_| | |_| |   <  __/\n"
+//                + "|____/ \\__,_|_|\\_\\___|\n";
+        String necroLogo =
+                  " ___    _                \n"
+                + "|   \\  | |   _____     ___    _  __     _____  \n"
+                + "| |\\ \\ | |  / __  \\  /   _|  | v __|  /  ___  \\ \n"
+                + "| | \\ \\| | |    __/  |  <_   |  /    |   |_|   |\n"
+                + "|_|  \\___|  \\_____|  \\____|  |__|     \\  ___  / \n";
+        System.out.println("Hello from\n" + necroLogo);
+        String tab = "      ";
+        String line = "------------------------------------------------------------";
```

Comment:
I think these 2 variables are already declared in the class fields? Maybe you can just reuse those variables?